### PR TITLE
Replace qplot with ggplot equivalent

### PR DESCRIPTION
**Pull request by vedhav**
Part of fix for [#458](https://github.com/insightsengineering/coredev-tasks/issues/458)

### CODE REVIEW

File: R/plot_with_settings.R [hunk 1]
```diff
@@ -191,7 +192,8 @@ plot_with_settings_ui <- function(id) {
 #'   ),
 #'   server = function(input, output, session) {
 #'     plot_r <- reactive({
-#'       ggplot2::qplot(x = 1:5, y = 1:5)
+#'       ggplot2::ggplot(NULL) +
```
Comment (kartikeyakirar) [inline R/plot_with_settings.R:195]: ggplot2::ggplot()  more conventional and concise .

Comment (vedhav) [reply R/plot_with_settings.R:195]: Agreed, Do you have any changes to this that you'd like?

File: R/plot_with_settings.R [hunk 2]
```diff
@@ -119,7 +119,8 @@ plot_with_settings_ui <- function(id) {
 #'   ),
 #'   server = function(input, output, session) {
 #'     plot_r <- reactive({
-#'       ggplot2::qplot(x = 1, y = 1)
+#'       ggplot2::ggplot(NULL) +
```
Comment (kartikeyakirar) [inline R/plot_with_settings.R:122]:
```suggestion
#'       ggplot2::ggplot() +
```

File: R/plot_with_settings.R [hunk 3]
```diff
@@ -119,7 +119,8 @@ plot_with_settings_ui <- function(id) {
...
+#'         ggplot2::geom_point(ggplot2::aes(x = 1, y = 1))
```
Comment (chlebowa) [inline R/plot_with_settings.R:123]:
```suggestion
#'       ggplot2::ggplot(faithful, ggplot2::aes(x = waiting, y = eruptions)) + ggplot2::geom_point()
```

File: R/plot_with_settings.R [hunk 4]
```diff
@@ -191,7 +192,8 @@ plot_with_settings_ui <- function(id) {
 #'   ),
 #'   server = function(input, output, session) {
 #'     plot_r <- reactive({
-#'       ggplot2::qplot(x = 1:5, y = 1:5)
+#'       ggplot2::ggplot(NULL) +
+#'         ggplot2::geom_point(ggplot2::aes(x = 1:5, y = 1:5))
```
Comment (chlebowa) [inline R/plot_with_settings.R:196]:
```suggestion
#'       ggplot2::ggplot(faithful, ggplot2::aes(x = waiting, y = eruptions)) + ggplot2::geom_point()
```

File: R/plot_with_settings.R [hunk 5]
```diff
@@ -227,7 +229,13 @@ plot_with_settings_ui <- function(id) {
 #'     )
 #'   ),
 #'   server = function(input, output, session) {
-#'     plot_r <- reactive(ggplot2::qplot(data = faithful, x = waiting, y = eruptions))
+#'     plot_r <- plot_r <- reactive(
+#'       ggplot2::ggplot(
+#'         data = faithful,
+#'         ggplot2::aes(x = waiting, y = eruptions)
+#'       ) +
```
Comment (chlebowa) [inline R/plot_with_settings.R:236]: Unless styler minds for some reason.

```suggestion
#'       ggplot2::ggplot(faithful, ggplot2::aes(x = waiting, y = eruptions)) +
```

**Review by chlebowa — APPROVED** [review]

**Pull request merged**